tests/vt_perf/baremetal.pm

```diff
@@ -0,0 +1,77 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: VT perf testsuite on Baremetal testing
+# Maintainer: James Wang <jnwang@suse.com>
+
+package baremetal;
+
+use warnings;
+use strict;
+use base "opensusebasetest";
+use Mitigation;
+use vt_perf_libs;
+use Utils::Backends 'use_ssh_serial_console';
+use bootloader_setup qw(grub_mkconfig change_grub_config add_grub_cmdline_settings remove_grub_cmdline_settings grep_grub_settings set_framebuffer_resolution set_extrabootparams_grub_conf);
+use ipmi_backend_utils;
+use power_action_utils 'power_action';
+use testapi;
+use utils;
+
+my $syspath         = '/sys/devices/system/cpu/vulnerabilities/';
+my $cpu             = get_var('CPU_FEATURE');
+my $vm_pool         = get_var('VM_POOL');
+my $vm_shares       = get_var('VM_SHARES');
+my $source_host     = get_var("SOURCE_HOSTNAME");
+my $dest_host       = get_var("DEST_HOSTNAME");
+my $subname         = "";
+my $cpu_1           = "";
```

Comment:
There are several variables are not used, The same issue happened on hvm_guest.pm and xen_guest.pm files. Could you remove the needless variables?

Reply:
OK